public/assets/css/own.css

```diff
@@ -0,0 +1,46 @@
+.bodytile {
+	background-color: white;
+	margin: 5px 15px 5px 15px;
+	color: #242a30;
+	transition: background-color 0.5s ease;
+}
+
+.bodytile:hover {
+	background-color: #00acac;
+}
+
+.bodytile-heading {
+	margin: 0 0 0 0;
+	padding: 5px 0 5px 0;
+}
+
+.bodytile-subheading {
+	margin: 0 0 0 0;
```

Comment:
Well, for playing around with it it's more handy, and also I kind of like that verbosity. I can change it though